.github/workflows/reusable-docker-ghcr.yml

```
@@ -59,14 +59,14 @@ jobs:
             org.opencontainers.image.revision=${{ github.sha }}
 
       - name: Add test tag
-        if: github.ref == format('refs/heads/{0}', inputs.develop_branch)
+        if: ${{ github.ref == format('refs/heads/{0}', inputs.develop_branch) && ! github.event.pull_request.head.repo.fork }}
```

```suggestion
        if: github.ref == format('refs/heads/{0}', inputs.develop_branch) && ! github.event.pull_request.head.repo.fork
```
You don't need the `${{}}` to jump into the GitHub Actions context as `if:` statements are already implicitly in that context, but it doesn't hurt to keep it.

---

Looks like in the docs the always include it now, so worth keeping for sure.